python/adjoint/objective.py

```diff
@@ -158,6 +158,11 @@ class EigenmodeCoefficient(ObjectiveQuantity):
         kpoint_func_overlap_idx: the index of the mode coefficient to return when
           specifying `kpoint_func`. When specified, this overrides the effect of
           `forward` and should have a value of either 0 or 1.
+        norm_dft_fields: the DFT fields obtained using `get_flux_data` from a
```

Comment:
Can re rename this to `subtracted_dft_fields`?

Reply:
Done.